### DISCUSSION TITLE
Proposal of Release and Tag

### DISCUSSION THREAD

**Discussion by ruomengh** [intel/confidential-cloud-native-primitives#244] [General]
Release:
Cadence: Quarterly
Naming convention: “v”+ version number, such as v0.3.0, v0.4.0

Tag:
Cadence: Bi-weekly by default,  also on demand for urgent fix
Naming convention: Incremental small version number based on last release. For example, last release is v0.3.0, following tags before next release is v0.3.1, v0.3.2…

**Comment by jyizheng** [comment]
LGTM

**Comment by wenhuizhang** [comment]
LGTM
